### ISSUE TITLE
what is overfitting? why wouldn’t the model cheating?

### ISSUE THREAD

**Issue by EmbraceLife** [EmbraceLife/shendusuipian#90]
### what is overfitting? why wouldn’t the model cheating?
40:03-41:40
[what is overfitting? why wouldn’t the model cheating?](https://ytcropper.com/cropped/Xf5c6ba4a250ea5)
How do we know the model is not cheating?
	- not learn the patterns to tell cricket from baseball 
	- but only member those specific objects in the images
How to avoid cheating
	- use validation set which your model doesn’t see when training
what  use validation set for?
	- use validation set to plot metrics to check how good model is fairly
Where is validation set?
	- automatically and directly baked into the DataBunch
	- to enforce the best practice, so it is impossible to not use it
目录 Lesson 1 知识点分解 #63
